cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -1311,11 +1311,26 @@ def name(self, value):
 
     @property
     def description(self):
-        return self._volume.get('description')
+        self.refresh()
+        labels = self._volume.get('labels')
+        if not labels or 'description' not in labels:
+            return ''
+        return labels.get('description', '')
 
     @description.setter
     def description(self, value):
-        raise NotImplementedError('Not supported by this provider.')
+        self.refresh()
```

Comment:
The reason is that to add or update labels, the setLabels method needs to provide with the latest fingerprint and new labels. Here is a relevant document on this topic.
https://cloud.google.com/compute/docs/labeling-resources

In commit e300d1b, I moved the refresh call to the end of the description setter, so that self._volume will hold the latest data of the volume resource (including the fingerprint) after a new label is set. I also removed the refresh call in the description getter, with the assumption that cloudbridge users only interact with GCE resources through cloudbridge.

In this way, two consecutive description setter calls will work naturally, and consecutive description setter and getter calls will work as well, without the need to call refresh in between.